build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -8,7 +8,8 @@
   </PropertyGroup>
   <ItemGroup>
     <MonoDocCopyItem Include="monodoc.dll" />
-    <MonoDocCopyItem Include="monodoc.dll.mdb" />
+    <MonoDocCopyItem Include="monodoc.pdb"  Condition="Exists ('monodoc.pdb')" />
```

Comment:
This won't actually work either; top-level `<ItemGroup/>` values are processed at file-load time, i.e. when `mono-runtimes.targets` is processed, and thus is quite possible that Mono won't have been built yet, and thus the `Condition` expressions here will *still* fail.

I think we'll need a "helper" target?